definitions.txt

```diff
@@ -19,11 +19,12 @@ Wa = watt * year
 
 EUR_2005 = [currency] = EUR = €_2005
 EUR_2000 = 0.94662 * EUR_2005 = €_2000
+EUR_2010 = 1.05677 * EUR_2005 = €_2010
```

Comment:
Are these inverted?

Inflation means a 2010 Euro buys _less_ than a 2005 Euro. For example, if you can buy some goods for EUR 100 in 2005, the same goods will have a price of EUR 105.677 in 2010. In 2010, EUR 100 is not enough to purchase these goods. In other words, 1.05677 EUR(2010) = 1 EUR(2005).

Reply:
thanks @khaeru for realizing that! those are inverted! I know what _inflation_ means, that would be critical for my job otherwise!!